src/views/layout.tsx

```diff
@@ -2,12 +2,27 @@ import React from "react";
 import { Flex, Grid, GridItem, Nav } from "@liftedinit/ui";
 import { AccountsMenu } from "features/accounts";
 import { NetworkMenu } from "features/network";
+import {
+  RiHome2Line,
+  RiServerLine,
+  RiFlowChart,
+  RiUserLine,
+  RiBankCardLine,
+} from "react-icons/ri";
+
+const navItems = [
+  { label: "Dashboard", path: "", icon: RiHome2Line },
+  { label: "Services", path: "settings", icon: RiServerLine },
+  { label: "Flows", path: "flows", icon: RiFlowChart },
+  { label: "Users", path: "users", icon: RiUserLine },
+  { label: "Billing", path: "account", icon: RiBankCardLine },
+];
```

Comment:
These have to be explicitly added now. They were in the Nav component before, but that component is now in the shared library and this config doesn't belong there.